raiden-dapp/src/components/SplashScreen.vue

```diff
@@ -1,29 +1,31 @@
 <template>
-  <v-container fluid class="splash-screen fill-height">
-    <v-row align="center" justify="center" no-gutters>
-      <v-col cols="4" lg="4" md="6" sm="8">
-        <div class="splash-screen__wrapper display-3">
-          <div class="splash-screen__logo-container">
-            <v-img
-              :src="require('../assets/logo.svg')"
-              min-width="50px"
-              class="splash-screen__logo"
-              aspect-ratio="1"
-              contain
-            />
-          </div>
-          <div class="splash-screen__app-name">
-            <div>
-              {{ name }}
-            </div>
-          </div>
+  <v-container class="splash-screen fill-height">
+    <v-row no-gutters justify="center">
+      <v-col cols="8" xl="2" lg="2" md="8" sm="8">
+        <div class="splash-screen__logo-container">
+          <v-img
+            :src="require('../assets/logo.svg')"
+            min-width="50px"
+            class="splash-screen__logo"
+            aspect-ratio="1"
+            contain
+          />
         </div>
-        <div class="font-weight-light text-center splash-screen__disclaimer">
-          {{ $t('splash-screen.disclaimer') }}
+      </v-col>
+      <v-col cols="8" xl="4" lg="5" md="8" sm="8">
+        <div class="splash-screen__app-name display-3">
+          {{ name }}
         </div>
-        <div class="font-weight-light text-center splash-screen__matrix_sign">
+      </v-col>
+      <v-col cols="8">
+        <div class="splash-screen__text font-weight-light text-center">
+          {{ $t('splash-screen.disclaimer') }}
+          <br />
```

Comment:
Do we really need the <br/> here? is it used to add space?

Reply:
I used them to add some space between the two paragraphs instead of having two divs for each and adding padding. Should I revert to the previous way?